### PR TITLE
Accessibility updates

### PR DESCRIPTION
**Pull request by lfarrell**
https://jira.lib.unc.edu/browse/BXC-2496
Does not address contrast issues. Probably needs a larger discussion on color choices.

### CODE REVIEW

File: static/css/sass/cdr-ui.scss [hunk 1]
```diff
@@ -664,8 +673,8 @@ p.spacing {
 }
 
 .item_container_count {
-  background-color: $unc-blue;
-  border: 1px solid $unc-blue;
+  background-color: #1A698C;
```
Comment (bbpennel) [inline static/css/sass/cdr-ui.scss:676]: Why not use a variable for these anymore?

File: static/css/sass/cdr_ui_styles.scss [hunk 2]
```diff
@@ -32,6 +33,18 @@ $box-shadow: inset 3px 3px 10px -1px $box-shadow-color, 0px 16px 22px -15px $box
   }
 }
 
+/* Class to display content for screen readers */
+.sr-only {
```
Comment (bbpennel) [inline static/css/sass/cdr_ui_styles.scss:37]: is this all standard for `sr-only` elements?

Comment (lfarrell) [reply static/css/sass/cdr_ui_styles.scss:37]: I stole it from bootstrap. It's what they use.

File: static/css/sass/cdr-ui.scss (hunk 1 above)
Comment (lfarrell) [reply static/css/sass/cdr-ui.scss:676]: $unc-blue still gets used in other places. This is a different, darker color, though it does get used a few times. I can add it as a variable

**Pull request merged**